packages/ui/alerts/alerts.jsx

```diff
@@ -190,6 +198,8 @@ export class Alerts extends React.PureComponent {
         if (typeof timeout === "number") {
           const id = message?.id;
 
+          this.onClearTickets(this.tickets);
```

Comment:
@ailincaibianca I'm not sure that this is ok. We would be clearing out `tickets` for `ids` that are not for this particular alert.
Imagine:
```
0 seconds
"push" -> "message-1-id" ->  ticket-1
"push" -> "message-2-id" -> ticket-2
2 seconds pass
"push" -> "message-2-id" -> ticket-3
now, because of `onClearTickets`
even message-1-id will be cleared through ticket-1 in `this.tickets`
```

I believe the original issue stems from the fact that `clearing` an alert depends on it's `id`, and that we keep a `timeout/ticket` for a particular `id` around even if the message was `pushed` again.

I believe we can solve this by keeping an `id` and the `ticket` in tickets as well.
```javascript
// at `push`:
const ticket = setTimeout(....);

// important do do before we push the new ticket to the tickets list
this.clearExistingTicket(id);

this.tickets.push({ id, ticket }); // push an object with both id and ticket




  clearExistingTicket = (id) => {
    const foundIndex = this.tickets.findIndex(entry => entry.id === id);

    if (foundIndex !== -1) {
      clearTimeout(foundIndex);
    }
  }

```